python/mxnet/sparse_ndarray.py

```diff
@@ -322,7 +317,7 @@ def _indices(self):
         raise Exception("unknown storage type " + stype)
 
     @property
-    def _indptr(self):
+    def indptr(self):
```

Comment:
I feel removing the prefix underscore to make indptr/indices "explicitly" accessible may not be a good idea, as they are attached to each specific type of sparse ndarrays while this is a general class for all types of sparse ndarrays. Keeping the underscore would remind users of using these properties with care and that one should use the universal aux_data interfaces for writing production code.